### DISCUSSION TITLE
Rebol/SQLite - Haiku Egg ?

### DISCUSSION THREAD

**Discussion by fvanzeveren** [Oldes/Rebol3#67] [General]
Hello

Is there any chance to have an egg to build the SQLite Extension on Haiku?

Thank you :)

François

**Comment by Oldes** [comment]
Try it with this commit: https://github.com/Siskin-framework/Rebol-SQLite/commit/3f3f5d27fc10823101cc9b1ebe3ebd6ccd343819

![image](https://user-images.githubusercontent.com/454805/234655224-e412a5a5-5137-4add-8f83-58883a0405ce.png)

**Comment by Oldes** [comment]
I must warn you, that the extension is not well tested, because I am not using it regularly... feel free to post any issues in its repository, if you found some.

**Comment by Oldes** [comment]
And btw... as it looks that you are a real Haiku user.. do you have any idea, where should be the best place to have a Rebol's home directory and where should be placed extensions? Currently there is used only current directory as a search path for modules and extensions. It may be modified setting `system/options/module-paths` (as I do in my `user.reb` file locally).

Ideally there should be some folder, where could be placed modules, so one could write just `import sqlite` instead of `import %path/to/sqlite.rebx` or use `needs: [sqlite]` in script headers.

**Comment by fvanzeveren** [comment]
Thanks oldes, the build is working and my first script using Rebol3/SQLite is working fine.
I will extensively test it in the next few days/weeks, by following https://www.sqlitetutorial.net/. I will post any issues I see.
For the modules directory, I would suggest `/boot/home/config/non-packaged/lib/` to store the rebx files.
And for the rebol.reb, I would use `/boot/home/config/settings/rebol/`
Regards
